spec/forms/prior_authority/steps/alternative_quotes/detail_form_spec.rb

```diff
@@ -36,7 +36,7 @@
   let(:service_type) { 'photocopying' }
 
   let(:file_upload) { instance_double(ActionDispatch::Http::UploadedFile, tempfile:, content_type:) }
-  let(:tempfile) { instance_double(File, size: 150) }
+  let(:tempfile) { Rails.root.join('spec/fixtures/files/test.png').open }
```

Comment:
Should this be a file or a path?

Reply:
It's a file - that `.open` at the end is some rails magic that is equivalent to `File.open(Rails.root.join('spec/fixtures/files/test.png'))`

Reply:
I meant should it expect a path 😄 Was trying to work out why the test failed